DearPyGui/src/core/AppItems/basic/mvButton.cpp

```diff
@@ -38,6 +38,33 @@ namespace Marvel {
 		auto styleManager = m_styleManager.getScopedStyleManager();
 		ScopedID id;
 
+		std::vector<int> pushedIDs;
```

Comment:
This could be static. The final version this could actually probably be an array since a widget should only have a set number of possible items that can be styled.

Reply:
done as array